### ISSUE TITLE
itest-flake: co-op channel fail during  update chan policy test teardown

### ISSUE THREAD

**Issue by Roasbeef** [lightningnetwork/lnd#5498]
At the very end of the test, for `btcd` backend (so far): 
```
    test_harness.go:88: Failed: (update channel policy): exited with error: 
        *errors.errorString unable to close channel: error while waiting for broadcast tx: wanted aedef3c01bb9e4c046acb0dddd6aaf65476a9a5c2cd7500ab41b06909ccc7e49, found 0 txs in mempool: []
        /home/travis/gopath/src/github.com/lightningnetwork/lnd/lntest/itest/lnd_test.go:242 (0xf32df8)
        	closeChannelAndAssertType: t.Fatalf("unable to close channel: %v", err)
        /home/travis/gopath/src/github.com/lightningnetwork/lnd/lntest/itest/lnd_test.go:217 (0xf3ae7f)
        	testUpdateChannelPolicy: return closeChannelAndAssertType(ctx, t, net, node, fundingChanPoint, false, force)
        /home/travis/gopath/src/github.com/lightningnetwork/lnd/lntest/itest/lnd_test.go:2058 (0xf3ae31)
        	testUpdateChannelPolicy: closeChannelAndAssert(ctxt, t, net, net.Alice, chanPoint, false)
        /home/travis/gopath/src/github.com/lightningnetwork/lnd/lntest/itest/test_harness.go:112 (0xeeb1ee)
        	(*harnessTest).RunTestCase: testCase.test(h.lndHarness, h)
        /home/travis/gopath/src/github.com/lightningnetwork/lnd/lntest/itest/lnd_test.go:11702 (0xf9ceaf)
        	TestLightningNetworkDaemon.func4: ht.RunTestCase(testCase)
        /home/travis/.gimme/versions/go1.16.3.linux.amd64/src/testing/testing.go:1193 (0x51c72f)
        	tRunner: fn(t)
        /home/travis/.gimme/versions/go1.16.3.linux.amd64/src/runtime/asm_amd64.s:1371 (0x471c01)
        	goexit: BYTE	$0x90	// NOP
=== CONT  TestLightningNetworkDaemon
    lnd_test.go:11710: Failure time: 2021-07-03 13:30:47.057
--- FAIL: TestLightningNetworkDaemon (806.43s)
    --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc (596.71s)
        --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=legacy (232.50s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=legacy/local_force_close_immediate_expiry (27.72s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=legacy/receiver_chain_claim (30.49s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=legacy/local_force_close_on-chain_htlc_timeout (26.10s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=legacy/remote_force_close_on-chain_htlc_timeout (25.80s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=legacy/local_chain_claim (45.21s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=legacy/remote_chain_claim (34.75s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=legacy/local_htlc_aggregation (37.98s)
        --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=anchors (343.77s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=anchors/local_force_close_immediate_expiry (44.10s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=anchors/receiver_chain_claim (38.92s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=anchors/local_force_close_on-chain_htlc_timeout (50.20s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=anchors/remote_force_close_on-chain_htlc_timeout (33.90s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=anchors/local_chain_claim (68.85s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=anchors/remote_chain_claim (57.60s)
            --- PASS: TestLightningNetworkDaemon/01-of-82/btcd/test_multi-hop_htlc/committype=anchors/local_htlc_aggregation (45.52s)
    --- PASS: TestLightningNetworkDaemon/02-of-82/btcd/sweep_coins (6.09s)
    --- PASS: TestLightningNetworkDaemon/03-of-82/btcd/recovery_info (14.40s)
    --- PASS: TestLightningNetworkDaemon/04-of-82/btcd/onchain_fund_recovery (29.89s)
    --- PASS: TestLightningNetworkDaemon/05-of-82/btcd/basic_funding_flow (66.18s)
        --- PASS: TestLightningNetworkDaemon/05-of-82/btcd/basic_funding_flow/carol_commit=legacy,dave_commit=legacy (7.02s)
        --- PASS: TestLightningNetworkDaemon/05-of-82/btcd/basic_funding_flow/carol_commit=legacy,dave_commit=tweakless (7.02s)
        --- PASS: TestLightningNetworkDaemon/05-of-82/btcd/basic_funding_flow/carol_commit=legacy,dave_commit=anchors (6.99s)
        --- PASS: TestLightningNetworkDaemon/05-of-82/btcd/basic_funding_flow/carol_commit=tweakless,dave_commit=legacy (6.99s)
        --- PASS: TestLightningNetworkDaemon/05-of-82/btcd/basic_funding_flow/carol_commit=tweakless,dave_commit=tweakless (6.98s)
        --- PASS: TestLightningNetworkDaemon/05-of-82/btcd/basic_funding_flow/carol_commit=tweakless,dave_commit=anchors (6.99s)
        --- PASS: TestLightningNetworkDaemon/05-of-82/btcd/basic_funding_flow/carol_commit=anchors,dave_commit=legacy (7.04s)
        --- PASS: TestLightningNetworkDaemon/05-of-82/btcd/basic_funding_flow/carol_commit=anchors,dave_commit=tweakless (6.99s)
        --- PASS: TestLightningNetworkDaemon/05-of-82/btcd/basic_funding_flow/carol_commit=anchors,dave_commit=anchors (6.97s)
    --- PASS: TestLightningNetworkDaemon/06-of-82/btcd/unconfirmed_channel_funding (7.32s)
    --- FAIL: TestLightningNetworkDaemon/07-of-82/btcd/update_channel_policy (74.45s)
```

**Comment by yyforyongyu** [comment]
Related from this [build](https://app.travis-ci.com/github/lightningnetwork/lnd/jobs/537190579),

```
test_harness.go:90: Failed: (update channel policy): exited with error: 
        *errors.errorString unable to get alice's balance: rpc error: code = Unknown desc = Ignoring outdated update (flags=00000001|00000000) for known chan_id=2205620325384192
        /home/travis/gopath/src/github.com/lightningnetwork/lnd/lntest/itest/lnd_channel_policy_test.go:340 (0x14cc99e)
        	testUpdateChannelPolicy: t.Fatalf("unable to get alice's balance: %v", err)
        /home/travis/gopath/src/github.com/lightningnetwork/lnd/lntest/itest/test_harness.go:114 (0x14b8f8e)
        	(*harnessTest).RunTestCase: testCase.test(h.lndHarness, h)
        /home/travis/gopath/src/github.com/lightningnetwork/lnd/lntest/itest/lnd_test.go:245 (0x1555394)
        	TestLightningNetworkDaemon.func4: ht.RunTestCase(testCase)
        /home/travis/.gimme/versions/go1.16.3.linux.amd64/src/testing/testing.go:1193 (0x523d0f)
        	tRunner: fn(t)
        /home/travis/.gimme/versions/go1.16.3.linux.amd64/src/runtime/asm_amd64.s:1371 (0x472201)
        	goexit: BYTE	$0x90	// NOP
```

**Comment by yyforyongyu** [comment]
fixed by #6825
